src/lib/swapper/swappers/ZrxSwapper/utils/constants.ts

```diff
@@ -24,3 +25,8 @@ export const ZRX_SUPPORTED_CHAINIDS = Object.freeze([
   KnownChainIds.PolygonMainnet,
   KnownChainIds.ArbitrumMainnet,
 ])
+
+export const ZRX_SUPPORTED_CHAIN_IDS: SupportedChainIds = {
+  sell: Object.keys(ZRX_SUPPORTED_CHAINIDS),
+  buy: Object.keys(ZRX_SUPPORTED_CHAINIDS),
```

Comment:
Not sure we want `Object.keys` on an array 👀 
<img width="317" alt="image" src="https://github.com/shapeshift/web/assets/17035424/dff4fb37-cabb-4bb6-b0ed-ff84d1c16a7a">

Reply:
Nice catch. The fact zrx is still working is likely due to another swapper supporting the same assets. The logic is a union of all supported swapper chains which explains why avalanche is still working on zrx.

Fixed in 13e529d525d114fa902c821323acbd28d70ea285 though